### PR TITLE
Plot screenshots using "s" key

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

File: pyaedt/generic/plot.py
Comment (Alberto-DM) [inline pyaedt/generic/plot.py:1053]:
```suggestion
            format = ".png"
```
if you use `os.path.splitext`

File: pyaedt/generic/plot.py
Comment (Alberto-DM) [inline pyaedt/generic/plot.py]:
```suggestion
                               root_name + datetime.now().strftime("%Y_%M_%d_%H-%M-%S") + format)
```
if you use `os.path.splitext`

File: pyaedt/generic/plot.py
Comment (Alberto-DM) [inline pyaedt/generic/plot.py:1050]:
```suggestion
            root_name, format = os.path.splitext(os.path.basename(export_image_path))
```
NOTE that format contains the dot (e.g. ".png")
Because: `root, ext = os.path.splitext(path)` is such as `path = root + ext`

**Review by Alberto-DM — APPROVED** [review]
Suggestion on using os.path to get the extension.
I am approving it, after the change you can merge it.

File: pyaedt/generic/plot.py
Comment (maxcapodi78) [inline pyaedt/generic/plot.py:1059]:
```suggestion
                path_image, "{}{}{}".format(root_name, datetime.now().strftime("%Y_%M_%d_%H-%M-%S"), format)
```

**Pull request merged**
